packages/node_modules/samples/browser-plugin-meetings/app.js

```diff
@@ -251,11 +251,14 @@ function joinMeeting(meetingId) {
 
   meeting.join(joinOptions)
     .then(() => {
+      window.meeting = meeting;
```

Comment:
No need to add this, there is already a function `getCurrentMeeting()` that returns you the meeting object